vendor.conf

```diff
@@ -103,7 +103,7 @@ github.com/googleapis/gax-go da06d194a00e19ce00d9011a13931c3f6f6887c7
 google.golang.org/genproto d80a6e20e776b0b17a324d0ba1ab50a39c8e8944
 
 # containerd
-github.com/containerd/containerd 89623f28b87a6004d4b785663257362d1658a729 # v1.0.0
+github.com/containerd/containerd 3fa104f843ec92328912e042b767d26825f202aa
```

Comment:
Should we add a note somewhere about this no longer being in sync with https://github.com/moby/moby/blob/2c8018f4bd7f48bf8f35770dea68f81b9591bb58/hack/dockerfile/binaries-commits#L7 ?